system-addon/test/unit/lib/ThemeFeed.test.js

```diff
@@ -66,6 +66,10 @@ describe("ThemeFeed", () => {
       feed.updateTheme({id: undefined});
       assert.calledOnce(dispatch);
       assert.equal(dispatch.firstCall.args[0].data.className, "");
+      dispatch.reset();
```

Comment:
let's split these out to their own `it()` so it better describes/comments what specifically we want to test